plugin-openshift/che-plugin-openshift-ext-client/src/main/resources/org/eclipse/che/ide/ext/openshift/client/OpenshiftLocalizationConstant.properties

```diff
@@ -86,3 +86,16 @@ invalid.openshift.project.name.error=Name must be a valid DNS name
 existing.project.name.error=A project with this name already exists
 
 invalid.che.project.name.error=Name must be a valid Che project name
+
+delete.project.action=Delete project
+delete.project.action.description=Delete project from OpenShift
+delete.project.dialog.title=Delete OpenShift project
+delete.project.without.app.label=Project {0} has not any apps. Deleting project {0}?
+delete.single.app.project.label=Deleting project {0}?
+delete.multiple.app.project.label=Project {0} has multiple apps: {1}. Deleting project {0} may cause problems with Che projects that are \
+  linked to it. You may need to reset them manually.
+delete.project.failed=Delete {0} project failed.
+delete.project.success=Project {0} successfully deleted.
+project.successfully.reset=Project {0} successfully reset.
+project.attribute.namespace.is.empty=Project attribute namespace is empty
```

Comment:
project.attribute.namespace.is.empty=Project attribute namespace is empty
Project {0} is not linked to OpenShift